### PR TITLE
Fix build with libbpfgo

### PR DESCRIPTION
**Pull request by yanivagman**
fix: https://github.com/aquasecurity/tracee/issues/686

### CODE REVIEW

**Review by itaysk — APPROVED** [review]
fine with me, but let's wait for @grantseltzer's approval before merging

**Comment by grantseltzer** [comment]
@yanivagman @itaysk 

How were you able to verify that this fixes the issue? When I do the same thing, push the change to a local branch, then update the libbpfgo go module locally I get a build failure on the `go get` command.

And assuming this does fix the issue should we include upgrading the libbpfgo module in tracee-ebpf in this PR?

**Comment by yanivagman** [comment]
@grantseltzer I changed tracee-ebpf/go.mod locally:
`replace github.com/aquasecurity/tracee/libbpfgo => ../libbpfgo`
and it now compiles correctly.

I think that upgrading libbpfgo module in tracee-ebpf should be done in a different commit.
In general, I prefer to split changes in tracee-ebpf and libbpfgo to different PRs

**Pull request merged**
